errors.go

```diff
@@ -0,0 +1,35 @@
+package ics
+
+const (
+	MalformedCalendarExpectedVCalendarError  = "malformed calendar; expected a vcalendar"
+	MalformedCalendarExpectedBeginError      = "malformed calendar; expected begin"
+	MalformedCalendarExpectedEndError        = "malformed calendar; expected a end"
+	MalformedCalendarExpectedBeginOrEndError = "malformed calendar; expected begin or end"
+
+	MalformedCalendarUnexpectedEndError             = "malformed calendar; unexpected end"
+	MalformedCalendarBadStateError                  = "malformed calendar; bad state"
+	MalformedCalendarVCalendarNotWhereExpectedError = "malformed calendar; vcalendar not where expected"
+
+	StartOrEndNotYetDefinedError = "start or end not yet defined"
+	PropertyNotFoundError        = "property not found"
```

Comment:
In order to be reliably compared these should be errors rather than strings.

Reply:
Agreed.

Reply:
(But technically not a blocker.)